source/snooty.txt

```diff
@@ -52,12 +52,8 @@ Installation
    This will only take effect in *new* terminal sessions.
 
 #. Select the :guilabel:`Extensions` pane by pressing Shift-Command(⌘)-X,
-   search for ``i80and.snooty``, and install it.
-
-   .. figure:: /images/vscode-install-snooty.png
-      :alt: VSCode extensions pane with snooty entered
-      :figwidth: 691px
-
+   search for ``Snooty``, and press **Install**.
```

Comment:
Suggestion: 
To be consistent with this item and the style guide, use the guilabel directive to refer to button/tab labels.

```suggestion
   search for ``Snooty``, and press :guilabel:`Install`.
```